setup.sh

```diff
@@ -1,69 +1,68 @@
-#!/bin/sh -ex
+#!/bin/bash
```

Comment:
Why switch to `bash`?
The only bashism there is `set -o pipefail`, and there isn't a single pipe...